cmd/kubectl-directpv/init_config_v1.go

```diff
@@ -18,6 +18,10 @@ package main
 
 import directpvtypes "github.com/minio/directpv/pkg/apis/directpv.min.io/types"
 
+const (
+	selectValueYes = "yes"
```

Comment:
if this contant is used at only one place, there is no need to declare

Reply:
It is used in two places